src/99-appendices/06-meg-file-formats.md

```diff
@@ -173,10 +173,17 @@ sub-control01/
             sub-control01_ses-001_task-rest_run-01_meg
             sub-control01_ses-001_task-rest_run-01_meg.json
             sub-control01_ses-001_task-rest_run-01_channels.tsv
-            sub-control01_ses-001_task-rest_run-01_markers.<mrk,sqd>
+            sub-control01_ses-001_task-rest_run-01[_acq-<label>]_markers.<mrk,sqd>
```

Comment:
Actually, according to the text in the [entities table appendix](https://bids-specification.readthedocs.io/en/stable/99-appendices/04-entity-table.html), the `acq` label must come before the `run` label :thinking: 

Could you change that @monkeyman192 ? (Sorry but I found it just now ...)

And it's another case where the validator is a bit too permissive, because when I tested it, it passed

Reply:
```suggestion
            sub-control01_ses-001_task-rest[_acq-<label>]_run-01_markers.<mrk,sqd>
```

Reply:
yep, sorry, been sidetracked! Will get onto it asap!